api/use_cases/__init__.py

```diff
@@ -0,0 +1,17 @@
+from api.share.responses import ResponseFailure, ResponseSuccess, Response
+from api.share.requests import ValidRequestObject
+
+
+class UseCase:
+    def execute(self, request_object: ValidRequestObject) -> Response:
```

Comment:
docstring? When I see this the first time, I don't understand what it's here for

Reply:
Ok